config/config.go

```diff
@@ -9,17 +9,21 @@ import (
 
 	"github.com/BurntSushi/toml"
 	"github.com/ardanlabs/conf"
+	"github.com/joho/godotenv"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	DefaultConfigPath = "config/config.toml"
+	DefaultEnvPath    = "config/.env"
```

Comment:
does this pick up every `*.env` file in the directory?

Reply:
no, just a file named .env in the config/ dir